### DISCUSSION TITLE
How to do Train (Newbie)

### DISCUSSION THREAD

**Discussion by Marbyun** [PaddlePaddle/PaddleOCR#11310] [Q&A]
Hi all! 
i'm right now do reasearch to do OCR for some id-cards (Indonesian) and want to training the model using KEI (RE/SER) i have read the docs for somedays and can't understand the step-by-step for train the model, specifically to prepared the dataset, how to make the label as question and the others as answer? can we use roboflow as work-around for the tools? 

thank you guys...

**Comment by benny-png** [comment]
https://chatgpt.com/share/03b349e3-f789-4492-ba9f-f5acca1197c8  check this out

**Comment by SWHL** [comment]
Suggest to read the [PaddleOCR docs](https://paddlepaddle.github.io/PaddleOCR/)
